javaServices/coreJavaServices/src/joliex/util/XmlUtils.java

```diff
@@ -158,6 +183,43 @@ public Value xmlToValue( Value request )
 			documentBuilderFactory.setSchema(null); // reset schema
 		}
 	}
+	
+	public Value xmlToPlainValue( Value request )
+		throws FaultException
+	{
+		try {
+			
+			Value result = Value.create();
+			InputSource src;
+			if ( request.isByteArray() ) {
+				src = new InputSource( new ByteArrayInputStream( request.byteArrayValue().getBytes() ) );
```

Comment:
So this is why the request type is any instead of just string.

What's the use case for raw? Shouldn't the client just convert the request to a string itself?